.github/workflows/os-zoo.yml

```diff
@@ -27,7 +29,7 @@ jobs:
       image: docker.io/library/alpine:${{ matrix.tag }}
     env:
       # https://www.openwall.com/lists/musl/2022/02/16/14
-      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare' || '' }}
+      EXTRA_CFLAGS: ${{ matrix.cc == 'clang' && '-Wno-sign-compare -Wno-switch-default' || '' }}
```

Comment:
Aehm, and why do we need the `-Wno-sign-compare` ?
Is this also just an old hack that seems to be necessary nowhere else ??

Reply:
quite possibly, but I would really rather address that in a separate pr, since we're just trying to get CI running again here

Reply:
sure, please take care to drop the -Wswitch-default in all branches, so eventually this can go away soon.

Reply:
will do, its marked for backport to all branches, so we should be ok

Reply:
no I don't mean this PR needs backport, since the OS Zoo CI is only started from the master branch by schedule
we need to address the -Wswitch-default in all branches Configure script.

Reply:
Either that or users moving to clang-18 will have to manually add -Wno-switch-default to their builds, but yes, we should fix that

Reply:
yes.